src/internal/components/dropdown/__integ__/dropdown-focus-trap.test.ts

```diff
@@ -13,7 +12,8 @@ interface PageSettings {
 
 function setupTest(settings: PageSettings, testFn: (page: DropdownPageObject) => Promise<void>) {
   return useBrowser(async browser => {
-    await browser.url(`#/light/dropdown/focus-trap?${qs.stringify(settings)}`);
+    await browser.url(`#/light/dropdown/focus-trap?settings=${JSON.stringify({ ...settings, loopFocus: true })}`);
```

Comment:
The settings object now needs to be complete, not partial. Unless the defaults handled differently in the page.